test/tape/index.js

```diff
@@ -0,0 +1,62 @@
+import test from 'tape';
+import thunk from 'redux-thunk';
+
+import { assertions, registerMiddlewares } from '../../src';
+import actions from '../testingData/actions';
+
+registerMiddlewares([thunk]);
+
+describe('tape', () => {
+  describe('t.plan', () => {
+    it('should finish the test without hanging', (done) => {
+      test('tape test', t => {
+        t.plan(1);
+
+        assertions.toDispatchActionsWithState(
+          { property: 'value' },
+          actions.actionCreatorWithGetState(),
+          actions.actionWithGetState({ property: 'value' }),
+          () => {
+            t.pass();
+            done();
+          },
+          t.fail
+        );
+      });
+    });
+  });
+
+  describe('t.plan', () => {
+    it('should finish the test without hanging', (done) => {
+      test('tape test', t => {
+        t.plan(1);
+
+        assertions.toDispatchActionsWithState(
+          { property: 'value' },
+          actions.actionCreatorWithGetState(),
+          actions.actionWithGetState({ property: 'value' }),
+          t.pass,
+          t.fail
+        ).then(done);
+      });
+    });
+  });
+
+  describe('t.end', () => {
+    it('should finish the test without hanging', (done) => {
+      test('tape test', t => {
+        assertions.toDispatchActionsWithState(
+          { property: 'value' },
+          actions.actionCreatorWithGetState(),
+          actions.actionWithGetState({ property: 'value' }),
+          () => {
+            t.pass();
+            t.end();
+            done();
+          },
+          t.fail
+        );
+      });
+    });
+  });
+});
```

Comment:
I'm a little stuck on why these tests won't pass, maybe you can give me some direction. I'm calling `done` in a couple different ways here to try and figure out what's going on, but only the first test ever passes. It seems like the other two are getting hung up on a promise somewhere, but I'm having trouble debugging. Test 1 and test 2 are the same, I was just trying to call done in different ways to figure out what's up.